tests/api/test_auth.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import json
+import requests
+import os
+
+from app import app, db
+from auth import AUTH0_DOMAIN, API_AUDIENCE
+
+class TestAuth(unittest.TestCase):
+  def setUp(self):
+    self.client = app.test_client()
+
+  def test_private_endpoint_no_auth(self):
+    response = self.client.get("/api/private")
+    self.assertEqual(response.status_code, 401)
+    json_response = json.loads(response.data.decode("utf-8"))
+
+    self.assertEqual(json_response["description"], "Authorization header is expected")
+  
+  @unittest.skipIf(not os.environ.get("AUTH0_CLIENT_ID") or not os.environ.get("AUTH0_CLIENT_SECRET"),
+                     "Cannot run test without AUTH0_CLIENT_ID and AUTH0_CLIENT_SECRET environment variables")
```

Comment:
Skip the test if the AUTH0_CLIENT_ID and AUTH0_CLIENT_SECRET env variables are `None` or empty strings.